Makefile

```
@@ -1,9 +1,23 @@
+export PATH := .venv/bin:$(PATH)
```

Probably better to use an absolute path to avoid issues if any subcommand decides to `cd` elsewhere.

```suggestion
export PATH := $(CURDIR)/.venv/bin:$(PATH)
```